src/model/Product/Product.js

```diff
@@ -11,9 +11,9 @@ const Product = () => {
     return await axios.get(url);
   }
   // api space
-  return {
-    fetchProducts
-  };
+  return (
+    <div>{product.id} {product.name}</div>
```

Comment:
why return view element. this is the api space and only return public functions

Reply:
got you but the file can be ignored atm, which is not in use